classes/Tutor.php

```diff
@@ -837,13 +856,134 @@ public static function create_database() {
 			PRIMARY KEY (withdraw_id)
 		) $charset_collate;";
 
+		$orders_table = "CREATE TABLE {$wpdb->prefix}tutor_orders (
+			id BIGINT(20) UNSIGNED AUTO_INCREMENT,
+			payment_id BIGINT(20) UNSIGNED COMMENT 'Unique payment id from payment gateway',
+			user_id BIGINT(20) UNSIGNED NOT NULL,
+			status ENUM('pending', 'completed', 'cancelled', 'refunded', 'partially-refunded') NOT NULL,
+			sub_total_price DECIMAL(13, 2) NOT NULL, -- price calculation based on course sale price
+			total_price DECIMAL(13, 2) NOT NULL,
+			order_price DECIMAL(13, 2) NOT NULL,
+			coupon_code VARCHAR(255),
+			discount_type ENUM('percentage', 'flat'),
+			discount_amount DECIMAL(13, 2),
+			discount_reason TEXT,
+			tax DECIMAL(13, 2) COMMENT 'Tax percentage',
+			tax_amount DECIMAL(13, 2),
```

Comment:
Maybe `taxable_amount`

Reply:
Actually, tax_amount will be thrown from the payment gateway, it means tax has already been paid. That's why we kept the name `tax_amount`